pipeline/measuring.py

```diff
@@ -414,3 +425,36 @@ def make_filter_bank(self, imsize, psf_fwhm):
         self._filter_bank = templates
         self._filter_psf_fwhm = psf_fwhm
 
+    def compare_measurement_to_thresholds(self, m):
+        """Compare measurement disqualifiers of a Measurements object to the thresholds set for 
+        this measurer object.
+
+        Inputs:
+          - m : a Measurements object to be compared
+
+        returns one of three strings to indicate the result
+          - "ok"     : All disqualifiers below both thresholds
+          - "bad"    : Some disqualifiers above mark_thresh but all 
+                       below deletion_thresh
+          - "delete" : Some disqualifiers above deletion_thresh
+        
+        """
```

Comment:
I will change the logic of this function soon following our Friday meeting, such that 
- you can pass `None` as a deletion_threshold, causing it to use the regular threshold as deletion thresh.
- you can pass an empty dict as deletion_threshold, causing it to do no deletion checking
- any keys missing in either dictionary will simply not be checked in that respective check (deletion or marking)